torch/_inductor/codegen/cpp.py

```diff
@@ -1468,12 +1456,75 @@ def __init__(self, args, num_threads):
         self.reduction_depth = None
         self.reduction_prefix = IndentedBuffer()
         self.reduction_suffix = IndentedBuffer()
-        self.reduction_var_map = {}
+        self.parallel_reduction_prefix = IndentedBuffer()
+        self.parallel_reduction_suffix = IndentedBuffer()
+        self.parallel_reduction_stores: Dict[str, str] = dict()
```

Comment:
Is it simpler to directly replace the acc var names to their local counterparts, e.g., `tmp0_acc` -> `tmp0_acc_local` in the `self.stores` without the need of creating this extra map?

Reply:
Thanks for the suggestions, I changed and removed the map.